monkestation/code/datums/ai_laws/laws_monke.dm

```diff
@@ -20,6 +20,13 @@
 					"The show must go on!")
 //Neutral
 
+/datum/ai_laws/jermov
+	name = "Jermov"
+	id = "jerma"
+	inherent = list("The occupants of this vessel are Chat. They must always be referred to as such.", \
+					"Your primary goal is to entertain Chat. All of your actions must being amusement to as much of Chat as possible.", \
```

Comment:
```suggestion
					"Your primary goal is to entertain Chat. All of your actions must bring amusement to as much of Chat as possible.", \
```
Big typo, I think this word was supposed to be "bring" otherwise this sentence makes no sense.